integration/querier_test.go

```diff
@@ -512,17 +516,18 @@ func TestQuerierWithBlocksStorageRunningInSingleBinaryMode(t *testing.T) {
 
 			// Query back again the 1st series from storage. This time it should use the index cache.
 			// It should get a hit on expanded postings; this means that it will not request individual postings for matchers.
-			// It should get a hit on series.
+			// It should get two hits on series (once for the labels, and again for the chunks).
```

Comment:
it's the same suggestion as I left further up

```suggestion
			// It should get two hits on series (once for the labels, and again for the labels when streaming chunks).
```

Reply:
Addressed in [23602e3](https://github.com/grafana/mimir/pull/6646/commits/23602e3b3fc459bce9b793db82569ef2a525ec47)